src/stores/artists-store.js

```diff
@@ -0,0 +1,16 @@
+import { observable, action } from 'mobx'
+
+class ArtistsStore {
+  @observable searchedArtist = ''
+  @observable foundArtists = []
+  @observable artistsInfo
+
+  @action
+  cleanArtists() {
+    this.foundArtists = []
```

Comment:
Hmm wydaje mi się to zbędnym krokiem i zamiast czyścić array i dodawać artystów pojedynczo lepiej po prostu nadpisać stary array nowym.

Reply:
should be done